### ISSUE TITLE
Can't sync with existing calendars

### ISSUE THREAD

**Issue by stemy2** [SimpleMobileTools/Simple-Calendar#921]
Hello,

I'm using DAVx5 to synchronize my agendas, but Simple Calendar can't. In caldav synchronization, the calendurs list is empty, even if the calendars exists.

**Comment by tibbi** [comment]
what app version are you using? Do you have any other calendar app on your device, where syncing works?

**Comment by stemy2** [comment]
I use Davx5 for synchronization and Etar for calendar, where the sync works.

**Comment by tibbi** [comment]
but what Simple Calendar app version are you using? It is at the bottom of the About page.

**Comment by stemy2** [comment]
6.5.1

**Comment by tibbi** [comment]
hm okay. Have you been asked the Calendar permission at toggling "CalDAV sync" for the first time? Double check in the device settings if the Calendar permission is granted.

**Comment by stemy2** [comment]
Yep, it is.

**Comment by ChelonianEgghead** [comment]
Having the same issue on Android 5.1 lollipop.

**Comment by mischlr** [comment]
Same issued Android 9.0
Simple Calendar Pro 6.5.4

If i use Simple Calendar (not Pro) 5.1.3 from Play Store it works like a charm.

**Comment by ChelonianEgghead** [comment]
It works with icsx5.

**Comment by inconspicuous-username** [comment]
I have the same issue as @mischlr mentioned.
The free (non-pro) version synchronizes perfectly, so I wanted to support the developers and bought the pro version. However, the pro version does not synchronize with my calendar of my google account.
I'm running android on 10, and the other calendars do sync

**Comment by angelo-fuchs** [comment]
I had a similar issue:
Installed Simple Calendar a while ago via FDroid.
Currently running version: 6.9.6 Pro
I had the Google Calendar deactivated
Then I installed Opencloud and DAVx5.
Synchronization worked.

In Simple Calendar:
The Calendar from DAVx5 was not showing in the CalDAV selection only the Google Account showed.
Tried reboot, but didn't change anything.

After I reactivated the Google Calendar app the List of Accounts was refreshed
Currently seems to work fine, but I'd rather not have to activate the Google Calendar App to run the Simple Calendar.
